newsrevenuehub/rev-engine
# [NRH-246] Page slug unique error message

#### What's this PR do?
Makes a few tweaks.
1. Primarily, makes sure to expose consistent error message for page-slug uniqueness violations. This is a violation against "deleted" pages too, as we soft delete them.
2. Moves error_messages to their own file in api app.
3. Incidentally makes a few pages to the Django admin donation page list view

#### How should this be manually tested?
Try to create a new page but use a duplicate slug. Notice the error message. Then delete a page, noting its slug, and try to create a new page with that "deleted" slug. Notice the same error message as above.

#### Do any changes need to be made before deployment to staging? production? (adding environment variables, for example)?
No
